.evergreen/publish.sh

```diff
@@ -7,23 +7,23 @@ set -o errexit  # Exit the script with error if any of the commands fail
 ############################################
 #            Main Program                  #
 ############################################
+RELATIVE_DIR_PATH="$(dirname "${BASH_SOURCE:-$0}")"
+. "${RELATIVE_DIR_PATH}/javaConfig.bash"
+
 RELEASE=${RELEASE:false}
 
 export ORG_GRADLE_PROJECT_nexusUsername=${NEXUS_USERNAME}
 export ORG_GRADLE_PROJECT_nexusPassword=${NEXUS_PASSWORD}
 export ORG_GRADLE_PROJECT_signingKey="${SIGNING_KEY}"
 export ORG_GRADLE_PROJECT_signingPassword=${SIGNING_PASSWORD}
 
-echo "Publishing snapshot with jdk11"
-export JAVA_HOME="/opt/java/jdk11"
-
 if [ "$RELEASE" == "true" ]; then
   TASK="publishArchives"
 else
   TASK="publishSnapshots"
 fi
 
-SYSTEM_PROPERTIES="-PjdkHome=/opt/java/jdk17 -Dorg.gradle.internal.publish.checksums.insecure=true -Dorg.gradle.internal.http.connectionTimeout=120000 -Dorg.gradle.internal.http.socketTimeout=120000"
+SYSTEM_PROPERTIES="-Dorg.gradle.internal.publish.checksums.insecure=true -Dorg.gradle.internal.http.connectionTimeout=120000 -Dorg.gradle.internal.http.socketTimeout=120000"
```

Comment:
Is `-PjavaVersion=${JAVA_VERSION}` needed here?

Reply:
No because JavaVersion is only used for testing purposes.